From 747c0cc4a6e41a3ef2bab046b3bbc273e80fdac6 Mon Sep 17 00:00:00 2001
From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
Date: Thu, 2 Mar 2023 14:08:42 +0530
Subject: [PATCH 16/18] Fix logic to display gvt prompt

Changes:
- improved string matching to handle both hex and decimal
- retained only SRIOV setting in string match

Tracked-On:
Signed-off-by: Suresh, Prashanth <prashanth.suresh@intel.com>

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index be32911..c21e0f4 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -188,7 +188,7 @@ function install_vm_manager() {
 }
 
 function ubu_enable_host_gvt(){
-    if [[ ! `cat /etc/default/grub` =~ "i915.enable_guc=0x7 udmabuf.list_limit=8192" ]] &&
+    if [[ ! `cat /etc/default/grub` =~ "i915.enable_guc="(0x)?0*"7" ]] &&
        [[ ! `cat /etc/default/grub` =~ "i915.enable_gvt=1" ]]; then
         read -p "Do you want to update the grub entry in '/etc/default/grub' for enabling GVT-g and GVT-d? [Y/n]" res
         if [ x$res = xn ]; then
-- 
2.40.0

